plugins/module_utils/network/common/utils.py

```diff
@@ -35,6 +35,7 @@
 import json
 
 from itertools import chain
+from yaml import safe_load
```

Comment:
Should we try using libyaml if available?
```try:
    # use C version if possible for speedup
    from yaml import CSafeLoader as SafeLoader
except ImportError:
    from yaml import SafeLoader```

Reply:
Fixed